### DISCUSSION TITLE
Add optional hyperlinks to port display text

### DISCUSSION THREAD

**Discussion by stolp** [seriousm4x/UpSnap#374] [Feature requests]
I love the port scanning functionality, which is giving me the availability to quickly discover if a service is running on a host.

I think it would be a great enhancement, if the text display can be optionally be extended to additionally point to a URI via a hyperlink.

So e.g. **HTTP (80)** would be rendered as a link to an URI which I can optionally add when configuring the port.

If an URI entry is given, a link should be used, else nothing changes.

This would allow users to reach their HTTP/HTTPS services directly from UpSnap.

This may be a duplicate of #222 as I just noticed.

**Comment by seriousm4x** [comment]
Will be available in the next release!
